ui/kotlinx-coroutines-javafx/src/JavaFxConvert.kt

```diff
@@ -0,0 +1,38 @@
+package kotlinx.coroutines.javafx
```

Comment:
Missing copyright. You can setup one in `Preferences -> Copyright profiles`:
`Copyright 2016-$today.year JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.`

Reply:
Configured and fixed.